besspin/cyberPhys/ui/hacker-kiosk/src/background.js

```diff
@@ -5,10 +5,12 @@ import { createProtocol } from 'vue-cli-plugin-electron-builder/lib'
 import installExtension, { VUEJS_DEVTOOLS } from 'electron-devtools-installer'
 const isDevelopment = process.env.NODE_ENV !== 'production'
 import {CanListener, CanNetwork} from '../../common/can';
+import zmq, { socket } from 'zeromq';
 
 
 let canQueue= [];
 
+// UDP Network
 let can_net = new CanNetwork(5002);
```

Comment:
Why does the kiosk need to listen to the CAN network?

Reply:
I don't think it should; it needs to listen to the special "hacked" CAN network that the GPS packets come in on, but other than that I don't think it uses CAN packets for anything.